hazelcast/src/hazelcast/client/MixedMap.cpp

```diff
@@ -1,297 +0,0 @@
-        }
-
-        std::auto_ptr<serialization::pimpl::Data>
-        MixedMap::putInternal(const serialization::pimpl::Data &keyData,
-                              const serialization::pimpl::Data &valueData, long timeoutInMillis) {
-            return proxy::IMapImpl::putData(keyData, valueData, timeoutInMillis);
-        }
-
-        void MixedMap::tryPutTransientInternal(const serialization::pimpl::Data &keyData,
-                                               const serialization::pimpl::Data &valueData,
-                                               int ttlInMillis) {
-            proxy::IMapImpl::tryPut(keyData, valueData, ttlInMillis);
-        }
-
-        std::auto_ptr<serialization::pimpl::Data>
-        MixedMap::putIfAbsentInternal(const serialization::pimpl::Data &keyData,
-                                      const serialization::pimpl::Data &valueData,
-                                      int ttlInMillis) {
-            return proxy::IMapImpl::putIfAbsentData(keyData, valueData, ttlInMillis);
-        }
-
-        bool MixedMap::replaceIfSameInternal(const serialization::pimpl::Data &keyData,
-                                             const serialization::pimpl::Data &valueData,
-                                             const serialization::pimpl::Data &newValueData) {
-            return proxy::IMapImpl::replace(keyData, valueData, newValueData);
-        }
-
-        std::auto_ptr<serialization::pimpl::Data>
-        MixedMap::replaceInternal(const serialization::pimpl::Data &keyData,
-                                  const serialization::pimpl::Data &valueData) {
-            return proxy::IMapImpl::replaceData(keyData, valueData);
-
-        }
-
-        void MixedMap::setInternal(const serialization::pimpl::Data &keyData,
-                                   const serialization::pimpl::Data &valueData,
-                                   int ttlInMillis) {
-            proxy::IMapImpl::set(keyData, valueData, ttlInMillis);
-        }
-
-        bool MixedMap::evictInternal(const serialization::pimpl::Data &keyData) {
-            return proxy::IMapImpl::evict(keyData);
-        }
-
-        std::auto_ptr<serialization::pimpl::Data>
-        MixedMap::executeOnKeyInternal(const serialization::pimpl::Data &keyData,
-                                       const serialization::pimpl::Data &processor) {
-            return proxy::IMapImpl::executeOnKeyData(keyData, processor);
-        }
-
-        std::auto_ptr<serialization::pimpl::Data>
-        MixedMap::submitToKeyDecoder(protocol::ClientMessage &response) {
-            return protocol::codec::MapExecuteOnKeyCodec::ResponseParameters::decode(response).response;
-        }
-
-        void
-        MixedMap::putAllInternal(const std::map<int, EntryVector> &entries) {
-            proxy::IMapImpl::putAllData(entries);
-        }
-
-        boost::shared_ptr<serialization::pimpl::Data>
-        MixedMap::toShared(const serialization::pimpl::Data &data) {
```

Comment:
This looks redundant since you already have the same method in ProxyImpl. 
And same method also can be deleted from ClientMapProxy